int32 drone_id

int8[] grid_ids

int8 opt #1 opt

std_msgs/Header header
IslandBox[] islands

float64 recent_attempt_time
float64 stamp
float64 allocation_cost

# only used for simulation
float32[] pos
float32[] vel
float32 yaw